config/config.exs

```diff
@@ -1,12 +0,0 @@
-# This file is responsible for configuring your application
-# and its dependencies with the aid of the Mix.Config module.
-import Config
-
-config :hammer,
-  backend:
-    {Hammer.Backend.ETS,
-     [
-       ets_table_name: :hammer_backend_ets_buckets,
-       expiry_ms: 60_000 * 60 * 2,
```

Comment:
what happened to this options?

Reply:
In `:ets` it's now called `:clean_period`. It's passed into `RateLimit.start_link`.